lutris/runners/steam.py

```diff
@@ -235,6 +239,17 @@ def get_steamapps_dirs(self):
                     dirs.append(path)
                 i += 1
 
+        # New Custom dirs
+        library_config = self.get_library_config()
+        if library_config:
+            i = 1
+            while "%s" % i in library_config:
+                path = library_config["%s" % i]["path"] + "/steamapps"
+                path = system.fix_path_case(path)
+                if path and os.path.isdir(path):
+                    dirs.append(path)
+                i += 1
```

Comment:
Why the need for a counter?
Could be replaced with:
```suggestion
            paths = [system.fix_path_case(entry["path"] + "/steamapps") for entry in library_config.values() if entry.get("path")]
            for path in paths:
                if os.path.isdir(path):
                    dirs.append(path)
```